packages/config-cli/src/cli/screenshot/index.ts

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
+import { BaseCommandLine } from '@basemaps/shared/build/cli/base.js';
+import 'source-map-support/register.js';
+import { CommandScreenShot } from './screenshot.js';
+
+export class ScreenshotCommandLine extends BaseCommandLine {
```

Comment:
lets call this `BasemapsConfig` as we will add more sub tools to it like the config import

`bmc` as the script name? or `basemaps-config` ?